test/unit_tests/api/test_processes.py

```diff
@@ -607,81 +618,69 @@ def allow(_: OIDCUserModel | None = None) -> bool:
     class ConfirmForm(FormPage):
         confirm: bool
 
-    @inputstep("unauthorized_resume", assignee=Assignee.SYSTEM, resume_auth_callback=disallow)
-    def unauthorized_resume(state):
-        user_input = yield ConfirmForm
-        return user_input.model_dump()
-
-    @inputstep("authorized_resume", assignee=Assignee.SYSTEM, resume_auth_callback=allow)
+    @inputstep("authorized_resume", assignee=Assignee.SYSTEM, resume_auth_callback=allow, retry_auth_callback=disallow)
     def authorized_resume(state):
         user_input = yield ConfirmForm
         return user_input.model_dump()
 
-    @inputstep("noauth_resume", assignee=Assignee.SYSTEM)
-    def noauth_resume(state):
-        user_input = yield ConfirmForm
-        return user_input.model_dump()
+    @step("next_step")
+    def next_step(state):
+        return {}
 
-    @workflow("test_auth_workflow", target=Target.CREATE)
+    @workflow("test_auth_workflow", target=Target.CREATE, authorize_callback=allow, retry_auth_callback=disallow)
     def test_auth_workflow():
-        return init >> noauth_resume >> authorized_resume >> unauthorized_resume >> done
-
-    with WorkflowInstanceForTests(test_auth_workflow, "test_auth_workflow"):
-        response = test_client.post("/api/processes/test_auth_workflow", json=[{}])
-        assert HTTPStatus.CREATED == response.status_code
-        process_id = response.json()["id"]
-        # No auth succeeds
-        response = test_client.put(f"/api/processes/{process_id}/resume", json=[{"confirm": True}])
-        assert HTTPStatus.NO_CONTENT == response.status_code
-        # Authorized succeeds
-        response = test_client.put(f"/api/processes/{process_id}/resume", json=[{"confirm": True}])
-        assert HTTPStatus.NO_CONTENT == response.status_code
-        # Unauthorized fails
-        response = test_client.put(f"/api/processes/{process_id}/resume", json=[{"confirm": True}])
-        assert HTTPStatus.FORBIDDEN == response.status_code
+        return init >> authorized_resume >> next_step >> done
```

Comment:
I don't see the `unauthorized_resume` step from before?

Reply:
added back